.azure-pipelines/linux.yaml

```diff
@@ -1,61 +1,38 @@
-resources:
-  containers:
-  - container: envoy-build-image
-    image: envoyproxy/envoy-build:latest
-
 trigger:
   branches:
     include:
     - 'master'
 
 jobs:
-# - job: build
-#   dependsOn: []
-#   pool:
-#     vmImage: 'Ubuntu 16.04'
-#   container: envoy-build-image
-#   steps:
-#     - checkout: self
-#       submodules: true
-#     - script: |
-#         bazel build //:envoy && \
-#         ./bazel-bin/envoy --version
-#       displayName: 'Build Envoy-OpenSSL'
-# - job: test
-#   dependsOn: []
-#   pool:
-#     vmImage: 'Ubuntu 16.04'
-#   container: envoy-build-image
-#   steps:
-#     - checkout: self
-#       submodules: true
-#     - script: |
-#         bazel test //test/common/... //test/extensions/...
-#       displayName: 'Test Envoy-OpenSSL'
-#  - job: test-envoy
-#    dependsOn: []
-#    pool:
-#      vmImage: 'Ubuntu 16.04'
-#    container: envoy-build-image
-#    steps:
-#      - checkout: self
-#        submodules: true
-#      - script: bazel test @envoy//test/...
-#        displayName: 'Test Envoy'
-  - job: verify
```

Comment:
Can we retain this job? This will ensure that we're not building with BoringSSL still.

Reply:
I'll look into it.